mkat_fpga_tests/test_cbf.py

```diff
@@ -776,7 +777,7 @@ def get_actual_phases():
         units = 'secs'
 
         aqf_plot_phase_results(no_chans, actual_phases, expected_phases,
-                               units, file_name, title, caption)
+                               units, file_name, title, caption, True)
```

Comment:
Perhaps remove 'True' to avoid automated test getting stuck here while waiting on matplotlib to close.